module/spring-boot-neo4j/src/main/java/org/springframework/boot/neo4j/autoconfigure/Neo4jAutoConfiguration.java

```diff
@@ -120,12 +139,11 @@ private void configurePoolSettings(Config.ConfigBuilder builder, Pool pool) {
 		builder.withMaxConnectionLifetime(pool.getMaxConnectionLifetime().toMillis(), TimeUnit.MILLISECONDS);
 		builder.withConnectionAcquisitionTimeout(pool.getConnectionAcquisitionTimeout().toMillis(),
 				TimeUnit.MILLISECONDS);
-		if (pool.isMetricsEnabled()) {
```

Comment:
I am not sure I like that change. If metrics vs. no metrics is no longer needed, then I think the `metricsEnabled` flag should go away and have metrics to be exported if an `ObservationRegistry` is available. That's what the other components are doing at the moment. 

Is there a reason to keep having a flag for this or was it just an attempt at migrating what we had before?